docs/querying/caching.md

```diff
@@ -82,3 +82,12 @@ Note that the task executor processes only support caches that keep their data l
 This restriction exists because the cache stores results at the level of intermediate partial segments generated by the
 ingestion tasks. These intermediate partial segments will not necessarily be identical across task replicas, so
 remote cache types such as `memcached` will be ignored by task executor processes.
+
+## Unsupported queries
```

Comment:
Some missing stuffs:

- `DataSourceMetadataQuery` is never cached.
- Any queries are not cached in brokers if `bySegment` is set in query context. Historicals don't seem to care about `bySegment` which I think is a bug. We should never cache when it's set.